package.json

```diff
@@ -33,10 +33,10 @@
   },
   "homepage": "https://github.com/sarcophagus-org/sarcophagus-v2-contracts#readme",
   "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.4",
```

Comment:
Migrated us to hardhat chai matchers from waffle https://hardhat.org/hardhat-chai-matchers/docs/migrate-from-waffle